CHEATSHEET.md

```diff
@@ -216,32 +221,33 @@ state.memory.store(0x6021f2, aaaa)
 Read Pointer to Pointer from Frame:
 
 ```python
-poi1 = new_state.se.any_int(new_state.regs.rbp)-0x10
-poi1 = new_state.se.any_int(new_state.memory.load(poi1, 8, endness='Iend_LE'))
+poi1 = new_state.solver.eval(new_state.regs.rbp)-0x10
+poi1 = new_state.solver.eval(new_state.memory.load(poi1, 8, endness='Iend_LE'))
 poi1 += 0x8
-ptr1 = (new_state.se.any_int(new_state.memory.load(poi1, 8, endness='Iend_LE')))
+ptr1 = (new_state.solver.eval(new_state.memory.load(poi1, 8, endness='Iend_LE')))
 ```
 
 Read from State:
 
 ```python
 key = []
 for i in range(38):
-    key.append(extractkey.se.any_int(extractkey.memory.load(0x602140+(i*4), 4, endness='Iend_LE')))
+    key.append(extractkey.solver.eval(extractkey.memory.load(0x602140+(i*4), 4, endness='Iend_LE')))
```

Comment:
`key = extractkey.mem[0x602140].int.array(38).resolved` :)

Reply:
you might not want to actually incorporate that though since this is just a cheatsheet. Maybe add it below as "this is equivalent"?